docs/5.api/rpc/transactions.md

```diff
@@ -260,8 +240,13 @@ Here is the exhaustive list of the error variants that can be returned by `broad
 
 - method: `tx`
 - params:
-  - `transaction hash` _(see [NearBlocks Explorer](https://testnet.nearblocks.io) for a valid transaction hash)_
-  - `sender account id`
+  - `tx_hash` _(see [NearBlocks Explorer](https://testnet.nearblocks.io) for a valid transaction hash)_
+  - `sender_account_id` _(used to determine which shard to query for transaction)_
+  - [Optional] `wait_until`: the required minimal execution level. Read more [here](/api/rpc/transactions#tx-status-result). The default value is `EXECUTED_OPTIMISTIC`.
+
+Transaction status request with `wait_until != NONE` will wait until the transaction appears in the chain.
+If the transaction does not exist, the method will wait until the timeout is reached.
+If you just need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.
```

Comment:
```suggestion
A Transaction status request with `wait_until != NONE` will wait until the transaction appears on the blockchain.
If the transaction does not exist, the method will wait until the timeout is reached.
If you only need to check whether the transaction exists, use `wait_until = NONE`, it will return the response immediately.
```